src/qml/editorwidgets/ValueRelation.qml

```diff
@@ -0,0 +1,118 @@
+import QtQuick 2.11
+import QtQuick.Controls 2.4
+
+
+import QtGraphicalEffects 1.0
+import QtQuick.Layouts 1.0
+import ".."
+import "../js/style.js" as Style
+
+import org.qfield 1.0
+import org.qgis 1.0
+
+Item {
+  id: valueRelation
+
+  RelationCombobox {
+    id: valueRelationCombobox
+    visible: !config['AllowMulti']
+    property var _relation: undefined
+
+    FeatureListModel {
+      id: featureListModel
+
+      currentLayer: qgisProject.mapLayer(config['Layer'])
+      keyField: config['Key']
+      displayValueField: config['Value']
+      addNull: config['AllowNull']
+      orderByValue: config['OrderByValue']
+    }
+  }
+
+  Rectangle{
+    id: valueRelationList
+
+    signal valueChanged(var value, bool isNull)
+
+    visible: config['AllowMulti']
+
+    height: Math.max( valueListView.height, itemHeight)
+    width: parent.width
+
+    property int itemHeight: 32 * dp
+
+    border.color: 'lightgray'
+    border.width: 1 * dp
+
+    FeatureCheckListModel {
+      id: listModel
+        attributeValue: value
+        currentLayer: qgisProject.mapLayer(config['Layer'])
+        keyField: config['Key']
+        displayValueField: config['Value']
+        addNull: config['AllowNull']
+        orderByValue: config['OrderByValue']
+    }
+
+    //the list
+    ListView {
+      id: valueListView
+      model: listModel
+      width: parent.width
+      height: Math.min( 5 * valueRelationList.itemHeight, valueListView.count * valueRelationList.itemHeight )
+      delegate: listComponent
+      focus: true
+      clip: true
+      highlightRangeMode: ListView.StrictlyEnforceRange
+    }
+
+    //list components
+    Component {
+      id: listComponent
+
+      Item {
+        id: listItem
+        anchors { left: parent.left; right: parent.right }
+
+        focus: true
+
+        height: Math.max( valueRelationList.itemHeight, valueText.height )
+
+        Row{
+          id: checkBoxRow
+          anchors { top: parent.top; left: parent.left }
+          height: listItem.height
+
+          CheckBox {
+            id: checkBox
+            width: parent.height
+            height: parent.height
+
+            checked: model.checked
```

Comment:
Evtl. chöntsch versueche, de RoleName z'ändere im model (bi roleNames()) uf `ItemChecked`
Nid sicher wohere das `model.` da zeigt...

```suggestion
            checked: ItemChecked
```

Reply:
Also du dänksch das isch de Grund, daser uf s'`FeatureListModel` gaht? 
Komisch, aber ich chas ja mal probiere...

Reply:
Gaht nöd. Es gaht immerno direkt is `data` vom `FeatureListModel`. Usserdem chund d'Mäldig:
"ReferenceError: ItemChecked is not defined" uf `ValueRelation.qml` (isch vorhär au scho mit `checked` cho)